src/borg/remote.py

```diff
@@ -17,7 +17,8 @@
 from .helpers import bin_to_hex
 from .repository import Repository
 
-RPC_PROTOCOL_VERSION = 2
+RPC_PROTOCOL_VERSION = 3
+RPC_PROTOCOL_DOWNGRADABLE_VERSION = 2
```

Comment:
maybe rather something simple for now.
we did not want to carry around too much heritage with borg anyway.